### PR TITLE
MNT: Upgrade versioneer 0.19 → 0.20

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
This will fix an LGTM.com error:
https://lgtm.com/projects/g/nipy/nibabel/snapshot/3b1ecf4fb3674052ae0297863aca98c9f84624e4/files/nibabel/_version.py?sort=name&dir=ASC&mode=heatmap#x8eb1df6955196f9d:1

### CODE REVIEW

File: nibabel/_version.py [hunk 1]
```diff
@@ -137,37 +136,28 @@ def git_get_keywords(versionfile_abs):
...
+                        keywords["date"] = mo.group(1)
     except EnvironmentError:
         pass
-    # CJM: Nibabel hack to ensure we can git-archive off-release versions and
-    # revert to old X.Y.Zdev versions + githash
-    try:
-        rel = runpy.run_path(os.path.join(os.path.dirname(versionfile_abs), "info.py"))
-        keywords["fallback"] = rel["VERSION"]
-    except (FileNotFoundError, KeyError):
-        pass
```
Comment (effigies) [inline nibabel/_version.py:163]: This hack is still required.

File: nibabel/_version.py [hunk 2]
```diff
@@ -221,16 +211,16 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
                     "full-revisionid": keywords["full"].strip(),
                     "dirty": False, "error": None,
                     "date": date}
-    # no suitable tags, so inspect ./info.py
+    # no suitable tags, so version is "0+unknown", but full hex is still there
     if verbose:
-        print("no suitable tags, falling back to info.VERSION or 0+unknown")
-    return {"version": keywords.get("fallback", "0+unknown"),
+        print("no suitable tags, using unknown + full revision id")
+    return {"version": "0+unknown",
```
Comment (effigies) [inline nibabel/_version.py:217]: Part of the same hack.

File: versioneer.py [hunk 3]
```diff
@@ -549,37 +548,28 @@ def git_get_keywords(versionfile_abs):
...
+                        keywords["date"] = mo.group(1)
     except EnvironmentError:
         pass
-    # CJM: Nibabel hack to ensure we can git-archive off-release versions and
-    # revert to old X.Y.Zdev versions + githash
-    try:
-        rel = runpy.run_path(os.path.join(os.path.dirname(versionfile_abs), "info.py"))
-        keywords["fallback"] = rel["VERSION"]
-    except (FileNotFoundError, KeyError):
-        pass
```
Comment (effigies) [inline versioneer.py:575]: Same hack.

File: versioneer.py [hunk 4]
```diff
@@ -633,16 +623,16 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
                     "full-revisionid": keywords["full"].strip(),
                     "dirty": False, "error": None,
                     "date": date}
-    # no suitable tags, so inspect ./info.py
+    # no suitable tags, so version is "0+unknown", but full hex is still there
     if verbose:
-        print("no suitable tags, falling back to info.VERSION or 0+unknown")
-    return {"version": keywords.get("fallback", "0+unknown"),
+        print("no suitable tags, using unknown + full revision id")
+    return {"version": "0+unknown",
```
Comment (effigies) [inline versioneer.py:629]: Once more.

File: versioneer.py [hunk 5]
```diff
@@ -961,37 +1045,28 @@ def git_get_keywords(versionfile_abs):
...
+                        keywords["date"] = mo.group(1)
     except EnvironmentError:
         pass
-    # CJM: Nibabel hack to ensure we can git-archive off-release versions and
-    # revert to old X.Y.Zdev versions + githash
-    try:
-        rel = runpy.run_path(os.path.join(os.path.dirname(versionfile_abs), "info.py"))
-        keywords["fallback"] = rel["VERSION"]
-    except (FileNotFoundError, KeyError):
-        pass
```
Comment (effigies) [inline versioneer.py:987]: And here.

File: versioneer.py [hunk 6]
```diff
@@ -1045,16 +1120,16 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
                     "full-revisionid": keywords["full"].strip(),
                     "dirty": False, "error": None,
                     "date": date}
-    # no suitable tags, so inspect ./info.py
+    # no suitable tags, so version is "0+unknown", but full hex is still there
     if verbose:
-        print("no suitable tags, falling back to info.VERSION or 0+unknown")
-    return {"version": keywords.get("fallback", "0+unknown"),
+        print("no suitable tags, using unknown + full revision id")
+    return {"version": "0+unknown",
```
Comment (effigies) [inline versioneer.py:1126]: And here.

**Comment by DimitriPapadopoulos** [comment]
Ouch, it's not really versioneer! Alternatively, I could just update the current patched 0.19 directly.

**Comment by effigies** [comment]
Will be updated in the next release.

**Pull request closed without merging**